app/main.py

```diff
@@ -4,0 +1,31 @@
+import json
+from os import path
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    config_path = path.join(path.dirname(__file__), "config.json")
```

Comment:
Ensure that the path to the JSON file is correct. According to the checklist, you should write the correct path to the JSON file.